internal/cli/server/chains/amoy.go

```diff
@@ -0,0 +1,70 @@
+package chains
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+var amoyTestnet = &Chain{
+	NetworkId: 80002,
+	Genesis: &core.Genesis{
+		Config: &params.ChainConfig{
```

Comment:
Can we reuse `AmoyChainConfig` defined in `params/config.go`?

Reply:
We will use `chains/amoy.go` predominantly as they are compatible with new CLI. `params/config.go` is still being used and for all networks. Will need to define a clear way to deprecate them but in a separate effort.